includes/RestApi/Themes/ThemeGeneratorController.php

```diff
@@ -446,12 +446,6 @@ private function validate_and_sanitize_flow_data( $flow_data ) {
 			$flow_data['theme']['template'] = Themes::get_flow_default_theme_slug( $current_flow );
 		}
 
-		if ( $flow_data['customDesign'] ) {
```

Comment:
Since we are not checking this should we have an alternative check with the new keys here?

Reply:
It is not required, as we are getting the color and font slug data from global styles.